main.go

```diff
@@ -753,13 +771,16 @@ func setupLogger() {
 
 }
 
-func initialiseSystem(arguments map[string]interface{}) error {
+func initialiseSystem() error {
 
 	// Enable command mode
 	for _, opt := range commandModeOptions {
-		v := arguments[opt]
-		if v != nil && v != false {
-			handleCommandModeArgs(arguments)
+		if opt != false {
```

Comment:
You should deduplicate these two, like `if opt != false || opt != "" {`

Reply:
I did have it like that but it fails go vet as a "suspect or"

Reply:
When I run go vet:
main.go:778: suspect or: opt != false || opt != ""
exit status 1

Reply:
But obviously it should work fine in this case.

Reply:
You're absolutely right - need to investigate further. This will likely be a go vet bug report :)

Reply:
On second thought, I'm retarded and vet is right.

`opt != false || opt != ""` is always `true`, because `opt == false && opt == ""` can never happen. Not sure what you wanted to do with this logic, but it's different from the old one.

Reply:
I've fixed it with some type assertion now